zk-tests/src/Cheatcodes.t.sol

```diff
@@ -190,4 +191,10 @@ contract ZkCheatcodesTest is Test {
 
         transactor.transact();
     }
+
+    function testBlockHash() public {
+        BlockhashCaller caller = new BlockhashCaller();
+        bytes32 hash = caller.getBlockhash(1);
+        require(hash == bytes32(0), "blockhash mismatch");
+    }
```

Comment:
This test can be part of `testZkPropagatedBlockEnv` in `Basic.t.sol` where the blocknumber can be hardcoded. We should check that the blockhash is zero both outside and within the zksync context.